srcpkgs/grub-btrfs/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'grub-btrfs'
+pkgname=grub-btrfs
+version=4.3
+revision=1
+build_style=gnu-makefile
```

Comment:
This package depends on `bash` and `grub`, no?

Reply:
`grub-utils` i would say

Reply:
I added grub-utils to the depends

Reply:
`41_snapshots-btrfs` is written in bash?

Reply:
I see. I added bash to the depends list